backend/dataall/modules/metadata_forms/services/metadata_form_service.py

```diff
@@ -37,6 +38,42 @@ def validate_create_field_params(data):
             raise exceptions.RequiredParameter('glossaryNodeUri')
 
 
+class MetadataFormAccessService:
```

Comment:
Curious is there a reason to implement a new way of checking user permissions rather than use the existing `ResourcePolicyService` and `has_resource_permission` decorator?

Reply:
When MF is created we don't make an entry in resource_permission_policy table. So, we can't check it there.
On the other hand, MF access rules are easy and rely on group|env|org-membership